dataframe-jdbc/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/db/MsSql.kt

```diff
@@ -0,0 +1,62 @@
+package org.jetbrains.kotlinx.dataframe.io.db
+
+import org.jetbrains.kotlinx.dataframe.io.TableColumnMetadata
+import org.jetbrains.kotlinx.dataframe.io.TableMetadata
+import org.jetbrains.kotlinx.dataframe.schema.ColumnSchema
+import java.sql.ResultSet
+import java.util.*
+import kotlin.reflect.KType
+import kotlin.reflect.full.createType
+
+/**
+ * Represents the MSSQL database type.
+ *
+ * This class provides methods to convert data from a ResultSet to the appropriate type for MSSQL,
+ * and to generate the corresponding column schema.
+ */
+public object MsSql : DbType("sqlserver") {
+    override val driverClassName: String
+        get() = "com.microsoft.sqlserver.jdbc.SQLServerDriver"
+
+    override fun convertSqlTypeToColumnSchemaValue(tableColumnMetadata: TableColumnMetadata): ColumnSchema? {
+        return null
+    }
+
+    override fun isSystemTable(tableMetadata: TableMetadata): Boolean {
+        val locale = Locale.getDefault()
+
+        fun String?.containsWithLowercase(substr: String) = this?.lowercase(locale)?.contains(substr) == true
+
+        val schemaName = tableMetadata.schemaName
+        val tableName = tableMetadata.name
+        val catalogName = tableMetadata.catalogue
+
+        return schemaName.containsWithLowercase("sys") ||
+            schemaName.containsWithLowercase("information_schema") ||
+            tableName.startsWith("sys") ||
+            tableName.startsWith("dt") ||
+            tableName.containsWithLowercase("sys_config") ||
+            catalogName.containsWithLowercase("system") ||
+            catalogName.containsWithLowercase("master") ||
+            catalogName.containsWithLowercase("model") ||
+            catalogName.containsWithLowercase("msdb") ||
+            catalogName.containsWithLowercase("tempdb")
```

Comment:
With this `containsWithLowercase` function, you're calling `.lowercase()`  5 times on the same string. Seems a bit unnecessary as this can be done just once when defining `catalogName`

Reply:
Need to refactor in databases and test on all of them, thanks